### ISSUE TITLE
YBD fails building x86_32 systems

### ISSUE THREAD

**Issue by anahuelamo** [devcurmudgeon/ybd#203]
Full log:

```
# ../ybd/ybd.py systems/build-system-x86_32.morph x86_32

...

16-04-06 00:00:08 [0/253/253] [stage1-binutils] Defined build system is autotools
16-04-06 00:00:08 [0/253/253] [stage1-binutils] Logging build commands to /src/ybdbase/artifacts/stage1-binutils.1096a097aa1ec888bef36760678dc5eb775a267f9d78a16fc0d0cb3d43569953.build-log 
16-04-06 00:00:08 [0/253/253] [stage1-binutils] Running configure-commands
16-04-06 00:00:08 [0/253/253] [stage1-binutils] Running command:
./configure --prefix="$PREFIX" --disable-nls --disable-werror \
  --build=$(sh config.guess) \
  --host=$(sh config.guess) \
  --target=$TARGET_STAGE1 \
  --with-sysroot=/nonexistentdir --with-lib-path="=$PREFIX/lib:=$PREFIX/lib64"
 
16-04-06 00:00:09 [0/253/253] [stage1-binutils] ERROR: command failed in directory /root/baserock/definitions:

 sh -c './configure --prefix="$PREFIX" --disable-nls --disable-werror \
  --build=$(sh config.guess) \
  --host=$(sh config.guess) \
  --target=$TARGET_STAGE1 \
  --with-sysroot=/nonexistentdir --with-lib-path="=$PREFIX/lib:=$PREFIX/lib64"
'
PROGRAM_REF=HEAD 16.13 (16.13 + 0 commits)
SHELL=/bin/sh
SOURCE_DATE_EPOCH=1432655321
TARGET=x86_32-baserock-linux-gnu
TARGET_STAGE1=x86_32-bootstrap-linux-gnu
TERM=dumb
TZ=UTC
USER=tomjon
USERNAME=tomjon
sh -c './configure --prefix="$PREFIX" --disable-nls --disable-werror \
  --build=$(sh config.guess) \
  --host=$(sh config.guess) \
  --target=$TARGET_STAGE1 \
  --with-sysroot=/nonexistentdir --with-lib-path="=$PREFIX/lib:=$PREFIX/lib64"
'

checking build system type... i686-pc-linux-gnu
checking host system type... i686-pc-linux-gnu
checking target system type... Invalid configuration `x86_32-bootstrap-linux-gnu': machine `x86_32-bootstrap' not recognized
configure: error: /bin/bash ./config.sub x86_32-bootstrap-linux-gnu failed

```

**Comment by devcurmudgeon** [comment]
were you running this on an x86_64 system? or a 32-bit VM? or something else?

**Comment by anahuelamo** [comment]
I was running in a x86_32 VM

**Comment by devcurmudgeon** [comment]
fixed, i believe, by b0ce7a6fd333ffdd0331efab044b52bd19fb36a0

**Comment by jjardon** [comment]
> fixed, i believe, by b0ce7a6

I do not think so, this has been fixed by commit 18ac1e9c39803df9418aead1f3c53e2d10c1b042

**Comment by devcurmudgeon** [comment]
Ah, thanks! :)
